src/main/java/by/fakeonliner/service/ShopService.java

```diff
@@ -1,4 +1,17 @@
 package by.fakeonliner.service;
 
+import by.fakeonliner.entity.Shop;
+import by.fakeonliner.repository.ShopRepository;
+
 public class ShopService {
+    private ShopRepository shopRepository = new ShopRepository();
```

Comment:
What use concrete class? May be make general interface

Reply:
fix